source/simulator/assemblers/stokes.cc

```diff
@@ -231,6 +232,7 @@ namespace aspect
 
               if (force != NULL)
                 data.local_rhs(i) += (force->rhs_u[q] * scratch.phi_u[i]
+                                      + double_contract(force->rhs_e[q],Tensor<2,dim>(scratch.grads_phi_u[i]))
                                       + pressure_scaling * force->rhs_p[q] * scratch.phi_p[i])
                                      * JxW;
```

Comment:
This means you will always assemble both the force vectors and the elasticity terms, even if only one of them is switched on (e.g. by setting `enable_elasticity` to true). At the moment, this works because the vectors are initialized with zeroes. But it means that, for example, if someone uses your new material model and also the force vector right-hand side, they will always have elasticity, if they want it or not (because you always fill the terms in the material model, no matter what the value of `enable_elasticity` is). So I don't think is is how it's intended to work. 

Maybe it would be better to have two if statements, one assembling the force terms 
`if (force != NULL && this->get_parameters().enable_additional_stokes_rhs)`
and one assembling the elasticity terms
`if (force != NULL && this->get_parameters().enable_elasticity)`.